velox/exec/MergeSource.h

```diff
@@ -23,6 +23,9 @@ class MergeExchange;
 
 class MergeSource {
  public:
+  static constexpr int32_t kMaxQueuedBytesUpperLimit = 32 << 20; // 32 MB.
```

Comment:
Does these need to be public? If not, let's move to private section? thanks!

Reply:
These are used by MergeExchange::addMergeSources() so need to be public. The other alternative is that we keep them private and handle the enforcement of upper and lower limit within MergeSource::createMergeExchangeSource(). This would abstract away the enforcement logic from the caller (MergeExchange) but this means the client loses explicit control. Either way is fine with me since we dont have other generic use cases yet for either alternative.
Please let me know if you want me to change the current implementation either way. Thanks